controllers/authpolicy_controller_test.go

```diff
@@ -1133,22 +1133,29 @@ var _ = Describe("AuthPolicy controller", func() {
 				fmt.Sprintf("AuthPolicy is conflicted by %[1]v/toystore: the gateway.networking.k8s.io/v1, Kind=HTTPRoute target %[1]v/toystore-route is already referenced by policy %[1]v/toystore", testNamespace),
 			), 30*time.Second, 5*time.Second).Should(BeTrue())
 		})
+
 		It("Invalid reason", func() {
-			const targetRefName, targetRefNamespace = "istio-ingressgateway", "istio-system"
+			var otherNamespace string
+			CreateNamespace(&otherNamespace)
+			// create a gateway in another namespace
```

Comment:
I'm gonna change this to create the policy in another namespace, not the gateway.